Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN ./build.sh configure_user
 
 # clean up scripts (it will get cached in layers, but oh well)
 WORKDIR /srv/
+RUN chown builder: /srv
```

Comment:
does this do anything considering /srv/ is almost always bind mounted over ?

Reply:
Its really useful for developement when you are doing ephemeral builds.